examples/research_projects/codeparrot/README.md

```diff
@@ -0,0 +1,66 @@
+# CodeParrot 🦜
+<p align="center">
+    <img src="images/code-highlighting-streamlit.png" alt="drawing" width="350"/>
+</p>
+
+## What is this about?
```

Comment:
Maybe add your GitHub handle here that indicates you are the author / maintainer of this script?